subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -177,11 +177,12 @@ Miscellaneous::
  * The internal `org.gradle.util` package is no longer imported by default.
 +
 Ideally you shouldn't use classes from this package, but, as a quick fix, you can add explicit imports to your build scripts for those classes.
+ * The `gradlePluginPortal()` repository no more infers artifact metadata from artifact files and now <<#rel5.0:gradle_plugin_portal_metadata,only uses Maven POMs>>.
```

Comment:
I think people won't know what that means. How about:

"The `gradlePluginPortal()` repository no longer looks for JARs without a POM by default"